great-idea/css/index.css

```diff
@@ -63,4 +65,207 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/*css for nav bar */
+nav {
+	display: flex;
+	padding-top: 30px;
+	padding-bottom: 20px;
+	width: 100%;
+	margin: 0 auto;
+	width: 100%;
+	align-items: center;
+}
+
+.menu {
+	width: 65%;
+	display: flex;
+	justify-content: space-between;
+}
+
+.menu a {
+	text-decoration: none;
+	color: #cfcfd4;
+	transition: ease-in-out 300ms;
+}
+
+.menu a:hover {
+	color: grey;
+}
+
+.menu-logo {
+	width: 25%;
+	display: flex;
+justify-content: flex-end;
+}
+
+/* style for header */
+.intro-home {
+	display: flex;
+	width: 100%;
+	margin-top: 50px;
+	padding-bottom: 60px;
+}
+
+div.intro {
```

Comment:
Here you call your class with `div.your-class {...}` while in the rest of your code you call it just with `.your-class {...}`.
Both ways are fine but for consistency is better to stick with one of those methods.